Projects/Foundations/Schemas/Docs-Topics/A/Alpha/Alpha-Data-Mine/alpha-data-mine-001-alpha-by-@rico4dev.json

```diff
@@ -19,7 +33,14 @@
         },
         {
             "style": "Success",
-            "text": "Collaborate with this data mine and report issues directly at the contributor’s repository."
+            "text": "Collaborate with this data mine and report issues directly at the contributor’s repository.",
+            "translations": [
+                {
+                    "language": "DE",
+                    "text": "Arbeiten Sie mit dieser Datenmine zusammen und melden Sie Probleme direkt an das Repository der Mitwirkenden.",
+                    "updated": 1641914649557
+                }
+            ]
         }
     ]
 }
```

Comment:
@harrellbm Is this warning sign a problem because there is something missing or  not?

Reply:
@This happens when the first translation for a file is added.  You can see the reason the red line is "changed" is to add a comma so the translations object can be appended to the end.  This one looks good!